.circleci/config.yml

```diff
@@ -2,14 +2,13 @@ version: 2
 jobs:
   linux_x86_64:
     docker:
-      - image: debian:stretch-backports
+      - image: debian:bullseye
```

Comment:
```suggestion
      - image: debian:buster
```

Reply:
Ubuntu 20.04 is based on Bullseye, not Buster!

Reply:
Part of the point with this build is to check that the release will build correctly with a low version requirement. I prefer to keep them low for the prebuild as it is mainly targeted to people who are unable to build for there specific system so a wide range of comparability is a benefit.

Reply:
Done